packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -2320,5 +2320,24 @@
     "country_selector_title": "Select your country:",
     "@country_selector_title": {
         "description": "Label written as the title of the dialog to select the user country"
-    }
+    },
+    "nutriscore_a": "Nutri-Score A",
```

Comment:
We're going to get "Nutri-Skor A" and the likes, which is a pain to maintain translation wise.

Reply:
For Nutri-Score, yes maybe…
But for Eco-score, in French we should say Éco-score :/
What would be the best approach?

Reply:
The branding for Eco-Score is not completely stabilized. Eco-Score won't hurt, I guess.

Reply:
Ok and what about NOVA, where "group" can also be translated?

Reply:
For nova, you mean the Ultra-processed foods attributes ? Do you want to concatenate NOVA to it ?

Reply:
The idea here is to provide a description that will be read by screen readers.
That's why I've provided a description for Nutriscore/Nova/Ecoscore, because now screen readers won't say anything